content/blog/2019-08-01-new-react-devtools.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Introducing the new React DevTools"
+author: [bvaughn]
+---
+We are excited to announce a new release of the React developer tools, available today in supported browsers!
```

Comment:
Nit: capitalize Developer Tools?

Reply:
Re: "supported browsers", can we be more specific? It doesn't tell you much since obviously it's unavailable in unsupported browsers — and if you don't know e.g. that FF is supported, this doesn't tell you either. Or we could remove the qualifier altogether if you don't want to go into detail.

Reply:
Sure. I had listed Chrome, Firefox, and Chromium Edge at first but then thought maybe it was too wordy and deleted it. I can re-add.